umbraco-cloud/set-up/project-settings/umbraco-cicd/KnownLimitationsAndConsiderations.md

```diff
@@ -18,7 +20,22 @@
 
 * Given the necessity to avoid changes in other environments, the lack of strict coordination among multiple teams or individuals working on the same project elevates the risk of conflicts.
 
-As we continue to prototype and gather insights from beta users, there are some known limitations and considerations to be aware of:
+**Usage of Private NuGet Feeds**
+
+It is possible to reference private NuGet feeds [by using the Secrets Management feature](../../private-nuget-feed.md) on Umbraco Cloud. 
+
+If you plan on using this feature, the first time you are setting up the private feed configuration has to be done manually. The guide describes how to correctly set it up. The initial setup is not possible through the CI/CD pipeline.
+
+This is because Umbraco Cloud collects the private NuGet credentials from your leftmost environment. If your leftmost environment does not have the credentials, trying to push the NuGet.config changes through CI/CD will cause restore problems.
```

Comment:
it makes better sense to have the text "The initial setup is not possible through the CI/CD pipeline." in 2nd paragraph as there is where an explanation why it is not working is 👍 
```suggestion
If you plan on using this feature, the first time you are setting up the private feed configuration has to be done manually. The guide describes how to correctly set it up. 

The initial setup is not possible through the CI/CD pipeline. This is because Umbraco Cloud collects the private NuGet credentials from your [left-most environment](https://docs.umbraco.com/umbraco-cloud/deployments/deployment). If your left-most environment doesn't have the credentials, then it will cause restore problems when trying to push the NuGet.config changes through CI/CD.
```